src/types.rs

```diff
@@ -412,6 +412,8 @@ pub struct RuntimeConfig {
 	pub max_cells_per_rpc: Option<usize>,
 	/// Time-to-live for DHT entries in seconds (default: 3600).
 	pub ttl: Option<u64>,
+	/// Threshold for the number of cells fetched from DHT,
+	pub threshold: usize,
```

Comment:
I think it would be good to have default value in case no option is specified in config, to avoid crashing LCs with old config. Like `dht_parallelization_limit`, for example.